test/tangy-form-item_test.html

```diff
@@ -127,6 +167,23 @@
           element.querySelector('tangy-form-item').shadowRoot.querySelector('tangy-checkboxes').dispatchEvent(new CustomEvent('next'))
           assert.equal(element.querySelector('tangy-form-item').open, false)
         })
+        test('should autostop after 2 answers.', () => {
+          const sleep = (milliseconds) => new Promise((res) => setTimeout(() => res(true), milliseconds))
+          const element = fixture('TangyFormItemAutoStopFixture');
+          element.newResponse()
+          let formItem = element.querySelector('tangy-form-item')
+          let el1 = formItem.shadowRoot.querySelector('[name=fruit_selection1]').value
+                  .find(input => (input.name === 'Tangerine'))
+          el1.value = 'on'
+          let helpers = new TangyFormItemHelpers(formItem)
+          let hideInputsUponThreshhold = (input) => helpers.hideInputsUponThreshhold(input)
```

Comment:
@rjsteinert - thanks a lot for your review - let's catch up on this later. A question - right now my implementation is barebones - the person building the form would have to add to the on-change  code to run this helper. Could we add this code to tangy-form-item? For example, if incorrect-threshold is set, then run `helpers.hideInputsUponThreshhold(input)` in the validate function in tangy-form-item? Is there a way to structure this addition to the validate function code so it doesn't become more spaghetti?